### PR TITLE
 [ES] localization of 'distributed system'

### PR DESCRIPTION
**Pull request by valeriahhdez**
Add [ES] localization of 'Distributed system'

### Describe your changes
This is a Spanish localization of the concept 'Distributed system' 

### Related issue number or link (ex: `resolves #issue-number`)
Issue #601 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/.wordlist.txt [hunk 1]
```diff
@@ -54,6 +54,7 @@ community
 Completed
 concept
 confiabilidad
+construido
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:57]:
```suggestion
```

File: content/es/.wordlist.txt [hunk 2]
```diff
@@ -76,6 +77,8 @@ DevOps
 DevSecOps
 DigitalOcean
 directly
+distribuido
+distribuidos
```
Comment (electrocucaracha) [inline content/es/.wordlist.txt:81]:
```suggestion
```

File: content/es/distributed-system.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Tradicionalmente, los sistemas implementarían un [escalamiento](/scalability/) vertical. 
+Esto es, cuando se añade más CPU o memoria a una máquina individual. 
+El escalamiento vertical consume mucho tiempo, requiere de un periodo de inactividad y alcanza su límite rápidamente. 
+
+## ¿Cómo ayuda?
+
+Los sistemas distribuidos permiten el [escalamiento horizontal](/horizontal-scaling/) (por ejemplo, añadir más nodos al sistema siempre que éste lo requiera). 
+Este proceso se puede automatizar, lo que permite al sistema manejar un incremento repentino de la carga de trabajo o del consumo de recursos. 
+Un sistema no distribuido está expuesto a los riesgos de falla porque cuando una máquina deja de funcionar, todo el sistema falla. 
+En cambio, un sistema distribuido puede ser contruido de tal manera que si una máquina deja de funcionar, el sistema completo puede seguir funcionando para producir los mismos resultados.
```
Comment (electrocucaracha) [inline content/es/distributed-system.md:30]:
```suggestion
En cambio, un sistema distribuido puede ser construido de tal manera
que si una máquina deja de funcionar, el sistema completo puede seguir funcionando para producir los mismos resultados. 
```

File: content/es/distributed-system.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+title: Sistema Distribuido
+status: Completed
+category: concepto
```
Comment (electrocucaracha) [inline content/es/distributed-system.md:4]:
```suggestion
category: Concepto
```

File: content/es/distributed-system.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+tags: ["architecture", "", ""]
+---
+
+## ¿Qué es?
+
+Un sistema distribuido es una colección de elementos computacionales autónomos conectados entre sí a través de una red que se muestra a los usuarios como un sistema único coherente.
```
Comment (electrocucaracha) [inline content/es/distributed-system.md:10]:
```suggestion
Un sistema distribuido es una colección de elementos computacionales autónomos
conectados entre sí a través de una red que se muestra a los usuarios como un sistema único coherente. 
```

File: content/es/distributed-system.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+title: Sistema Distribuido
+status: Completed
+category: concepto
+tags: ["architecture", "", ""]
+---
+
+## ¿Qué es?
+
+Un sistema distribuido es una colección de elementos computacionales autónomos conectados entre sí a través de una red que se muestra a los usuarios como un sistema único coherente. 
+Conocidos usualmente como [nodos](/es/nodes/), estos elementos pueden ser dispositivos de hardware (por ejemplo, computadoras o teléfonos móviles) o procesos de software.
```
Comment (electrocucaracha) [inline content/es/distributed-system.md:11]:
```suggestion
Conocidos generalmente como [nodos](/es/nodes/), estos elementos pueden ser dispositivos de hardware (por ejemplo, computadoras o teléfonos móviles) o procesos de software. 
```

File: content/es/distributed-system.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+Los nodos están programados para lograr un objetivo común e intercambian mensajes a través de la red para poder colaborar. 
+
+## Problema que aborda
+
+Una gran cantidad de aplicaciones modernas son tan grandes que necesitan de supercomputadoras para poder operar. 
+Ninguna computadora es lo suficientemente poderosa para alojar toda una aplicación. 
+Sin embargo, cuando se conectan varias computadoras el poder de computación se vuelve casi ilimitado. 
+Sin computación distribuida, no sería posible construir muchas de las aplicaciones que utilizamos hoy en día. 
+
+Tradicionalmente, los sistemas implementarían un [escalamiento](/scalability/) vertical.
```
Comment (electrocucaracha) [inline content/es/distributed-system.md:21]:
```suggestion
Tradicionalmente, los sistemas implementarían un [escalamiento](/es/scalability/) vertical. 
```

File: content/es/distributed-system.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+Sin computación distribuida, no sería posible construir muchas de las aplicaciones que utilizamos hoy en día. 
+
+Tradicionalmente, los sistemas implementarían un [escalamiento](/scalability/) vertical. 
+Esto es, cuando se añade más CPU o memoria a una máquina individual. 
+El escalamiento vertical consume mucho tiempo, requiere de un periodo de inactividad y alcanza su límite rápidamente. 
+
+## ¿Cómo ayuda?
+
+Los sistemas distribuidos permiten el [escalamiento horizontal](/horizontal-scaling/) (por ejemplo, añadir más nodos al sistema siempre que éste lo requiera). 
+Este proceso se puede automatizar, lo que permite al sistema manejar un incremento repentino de la carga de trabajo o del consumo de recursos.
```
Comment (electrocucaracha) [inline content/es/distributed-system.md:28]:
```suggestion
Este proceso se puede automatizar, lo que permite al sistema manejar un incremento repentino de la carga de trabajo o del consumo de recursos.

```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Además de estos cambios, sería necesario renombrar el archivo por `distributed-systems.md`

**Review by electrocucaracha — APPROVED** [review]

**Comment by ramrodo** [comment]
/lgtm
@92nqb @valeriahhdez could you review/approve this PR, please?

**Review by krol3 — APPROVED** [review]
lgtm, Thanks @valeriahhdez !

**Pull request merged**